sqlmesh/core/scheduler.py

```diff
@@ -246,6 +247,7 @@ def run(
         circuit_breaker: t.Optional[t.Callable[[], bool]] = None,
         deployability_index: t.Optional[DeployabilityIndex] = None,
         auto_restatement_enabled: bool = False,
+        is_run_command: bool = False,
```

Comment:
The variable names feels like an abstraction leak. Shouldn't this be `run_environment_statements: bool`